src/table/table.scss

```diff
@@ -122,265 +249,341 @@
+        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      }
 
-            > .iui-icon {
-              width: $iui-icons-default;
-              height: $iui-icons-default;
-            }
-          }
+      &:hover {
+        @include themed {
+          background: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-5)
+          );
         }
+      }
 
-        > .iui-filter,
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          height: $iui-icons-default;
-          width: $iui-icons-default;
-          display: flex;
-          justify-content: center;
-          align-items: center;
-          visibility: hidden;
+      + .iui-selected {
+        border-bottom: 0;
+        padding-bottom: $iui-xxs;
+        padding-top: 0;
+        @include themed {
+          border-top: 1px solid
+            rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
         }
+      }
 
-        > .iui-filter.iui-active {
-          visibility: visible;
-  
-          > .iui-icon {
-            @include themed {
-              fill: t(iui-icons-color-primary);
-            }
-          }
+      &:last-child {
+        padding-bottom: 1px;
+        @include themed {
+          border-bottom: 1px solid t(iui-color-foreground-primary);
         }
+      }
 
-        &:hover {
-          @include themed {
-            background-color: t(iui-color-background-4);
-          }
-
-          > .iui-filter,
-          > .iui-sort,
-          > .iui-sort > .iui-icon-wrapper {
-            visibility: visible;
-          }
-        }
+      &:first-child {
+        border-bottom: 0;
+        padding-bottom: $iui-xxs;
       }
 
-      &.iui-active-sort {
+      // Last selected item in a row.
+      + :not(.iui-selected) {
+        padding-top: 0;
+        margin-left: 0;
+        margin-right: 0;
         @include themed {
-          background-color: t(iui-color-background-4);
+          border-top: 1px solid t(iui-color-foreground-primary);
         }
+      }
 
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          visibility: visible;
+      // Added by Jon, not fully tested
+      + .iui-expanded-content {
+        @include themed {
+          border-color: t(iui-color-foreground-primary);
         }
       }
     }
 
-    &.iui-tables-row-active {
-      @include themed {
-        border: solid 1px t(iui-color-foreground-primary);
-        background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
-      }
+    &:last-child {
+      padding-bottom: $iui-xxs;
+      border-bottom: 0;
     }
+    // #endregion Selection
 
-    &.iui-tables-row-new {
+    // #region Statuses & states
+    &.iui-new {
       font-weight: $iui-font-weight-semibold;
-
-      > .iui-tables-filename {
-        position: relative;
-      }
-
-      > .iui-tables-filename::before {
+      
+      > .iui-main-column::before {
```

Comment:
What's "main column"?

And where's "secondary column"?

Reply:
The main column is whatever column we want to use to display the new file dot, file uploading progress, or file status.  I would argue that this column can not be disabled in the column manager.
![Screen Shot 2021-05-18 at 4 24 55 PM](https://user-images.githubusercontent.com/849817/118718734-721b1e00-b7dc-11eb-8895-935e55dab272.png)

Reply:
Makes sense. I will leave this comment open for now, for visibility

Reply:
This is soooo specific..
And I agree class name is really confusing...

Reply:
I am open to renaming the class if you have a better suggestion.  I've done some cleanup on the contents of the `.iui-main-column`:
- `.iui-filetype` handles the icon in the front rather than targeting `.iui-icon` or all `svg`.
- `div.iui-status` handles the spinner and/or status icons & center aligns them with the sort arrow in the header.